docs/docs/how-tos/use-gpus.mdx

```diff
@@ -0,0 +1,105 @@
+
+Select an appropriate profile and click "Start".
+
+### Understanding GPU setup on the server.
+The following steps describe how to get CUDA-related information from the server.
+1. Once your server starts, it will redirect you to a JupyterLab home page.
+2. Click on the **"Terminal"** icon.
+3. Run the command `nvidia-smi`. The top right corner of the command's output should have the highest supported driver.
+    ![nvidia-smi-output](/img/how-tos/nvidia-smi-output.png)
+
+    If you get `nvidia-smi: command not found`. You are most likely on a non-GPU server. Shutdown your server, and start up a GPU-enabled server.
+
+## 2. Creating environments
+
+By default, `conda-store` will build CPU-compatible packages. To build GPU-compatible packages, we the following.
+### Build a GPU-compatible environments
+By default, conda-store will build CPU-compatible packages. To build GPU-compatible packages, we have a couple of options:
+1. **Creating config using PyTorch documentation**:
+
+    You will need to check PyTorch documentation to [select a supported version of CUDA from the support matrix](https://pytorch.org/get-started/locally/) to determine the latest override version for `CONDA_OVERRIDE_CUDA`.
+    Select the following options to get the latest CUDA version:
+    - PyTorch Build = Stable
+    - Your OS          = Linux
+    - Package          = Conda
+    - Language         = Python
+    - Compute Platform = 12.1 (Choose the latest supported platform)
+
+    ![pytorch-linux-conda-version](/img/how-tos/pytorch-linux-conda-version.png)
+
+    The command `conda install` from above is:
+    ```
+    conda install pytorch torchvision torchaudio pytorch-cuda=12.1 -c pytorch -c nvidia
+    ```
+    The corresponding yaml config is:
+    ```
+    channels:
+      - pytorch
+      - nvidia
+      - conda-forge
+    dependencies:
+      - pytorch
+      - pytorch-cuda==12.1
+      - torchvision
+      - torchaudio
+      - ipykernel
+    variables: {}
+    ```
+    You can use **GUI <-> YAML** Toggle to edit the config.
+
+2. **Build a GPU-compatible environment in conda-store using `CONDA_OVERRIDE_CUDA`**:
+
+    Conda-store provides an alternate mechanism to enable GPU environments via the setting of an environment variable as explained in the [conda-store docs](https://conda.store/conda-store-ui/tutorials/create-envs#set-environment-variables).
+    While creating a new config, click on the **GUI <-> YAML** Toggle to edit yaml config.
+    ```
+    channels:
+      - pytorch
+      - nvidia
+      - conda-forge
+    dependencies:
+      - pytorch
+      - ipykernel
+    variables:
+      CONDA_OVERRIDE_CUDA: "12.1"
+    ```
+
+    Replace this section with an override for `CONDA_OVERRIDE_CUDA` to tell the conda-store to build a GPU-compatible environment instead of the default CPU-compatible one.
+    ![pytorch-cuda-override.png](/img/how-tos/pytorch-cuda-override.png)
+    :::note
```

Comment:
Be sure to only keep the first mention of the max cuda version (after the two sections are swapped).